### DISCUSSION TITLE
Vote on new Terms of Reference

### DISCUSSION THREAD

**Discussion by bharveyTX** [CC-in-the-Cloud/Admin#126] [Announcements]
We have created a major revision to our Terms of Reference.

Please review the new [TOR](https://github.com/CC-in-the-Cloud/General/blob/Working/iTC%20Admin%20Docs/iTC-ToR.pdf) and cast your vote to accept or reject the new terms.

Please cast your vote by filling out the form below:(https://docs.google.com/forms/d/e/1FAIpQLSf6UOokOm3m3IVKPvSvh5OwaQOcz1RsUKQi_ABzkvOl-pCAAg/viewform?vc=0&c=0&w=1&flr=0).

If you choose to abstain, please provide a comment for record keeping. 

The vote will remain open for two weeks until December 20th, 2023 at 24:00 UTC.

**Comment by jgb1128** [comment]
The votes are in:
10 unique valid votes
1 dup
1 invalid vote (email address and name not on list)

No: 0
Abstain: 0
Yes: 10

So the TOR passed. 

I will upload the anonymized votes.
There were also minor editorial catches which we will integrate before posting. I will share as well as a separate issue.
